modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -33,3 +35,30 @@ function acquia_cms_headless_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Install and update new headless roles.
+ */
+function acquia_cms_headless_update_8001() {
+  $role = Role::load('headless');
+  if ($role instanceof Role) {
+    $role->set('label', 'Headless Administrator');
+    $role->save();
```

Comment:
@deepakmishra2 There's no need to call `user_role_grant_permissions` function instead we can directly call `grantPermission()` function i.e
```suggestion
  $permissions = [
    'view any unpublished content',
    'view latest version',
    'view all revisions',
  ];
  foreach($permissions as $permission) {
    $role->grantPermission($permission);
  }
  $role->trustData()->save();
```